CHANGES.md

```diff
@@ -2,6 +2,16 @@
 
 This file is only intended to serve the developers of this repository and of the releases. This provides an overview of the releases that have been made using this release pipeline.
 
+## April 2024: Security releases for 1.41.1
+
+- [MediaWiki release announcement: 1.41.1](https://lists.wikimedia.org/hyperkitty/list/wikitech-l@lists.wikimedia.org/thread/V3WXEPXV2DU6WTVEKK4XHW4QXD5OFKD7/)
+- [MediaWiki full release notes](https://www.mediawiki.org/wiki/Release_notes/1.41#MediaWiki_1.41.1)
+- [Wikibase Suite Release task T361228](https://phabricator.wikimedia.org/T361228)
+
+| Suite Version | Version Info | Date available | Release task |
+| --- | --- | --- | --- |
+| wmde.20 ([spec](https://github.com/wmde/wikibase-release-pipeline/blob/wmde.20/variables.env)) | MediaWiki (1.41.1, PHP 8.2.15, debian/apache 2.4.57-2), WDQS (0.3.137) | XX April 2024 | [T361228](https://phabricator.wikimedia.org/T361228) |
```

Comment:
Re. `XX April 2024`: Understanding that we don't know the value until we actually merge, but this should be populdated. The wide.17 note below also is `XX March 2024` and should be updated to the actual release date.

Reply:
Corrected `XX March`, guessed on `XX April`